config/envs/prod/shield.settings.yml

```diff
@@ -0,0 +1,14 @@
+credential_provider: shield
```

Comment:
Don't we want this disabled on prod? Why are we tracking config for it?

Reply:
Oooh yeah definitely don't want it on prod. we aren't even using it at all, maybe we kill it off